### ISSUE TITLE
Accessibility: Missing alt tags on images

### ISSUE THREAD

**Issue by bcl1713** [bcl1713/ChoreQuest#50]
Some of the images in the application are missing `alt` tags. This is an accessibility issue for users who rely on screen readers. All images should have descriptive `alt` tags.
